test/dotcom_web/controllers/schedule/timetable_controller_test.exs

```diff
@@ -224,11 +224,17 @@ defmodule DotcomWeb.ScheduleController.TimetableControllerTest do
     end
   end
 
-  describe "trip_messages/2" do
-    test "returns proper messages for a CR Franklin train running via Fairmount" do
+  describe "trip_messages/3" do
+    setup do
+      # hardcoded date before the rating switchover that'll change the relevant trip numbers
```

Comment:
Honestly, this is a good comment. I do have a hazy idea about a way to solve the problem raised in the comment, but this is high priority enough not to need to be blocked on it. I wouldn't have even been thinking about it without the comment though.